docs/gateway/nas.md

```diff
@@ -0,0 +1,48 @@
+# Minio NAS Gateway [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+Minio Gateway adds Amazon S3 compatibility to NAS storage.
+
+## Run Minio Gateway for NAS Storage
+
+### Using Docker
+```
+docker run -p 9000:9000 --name nas-s3 \
```

Comment:
We need to write a bit more documentation here can you check with @abperiasamy about this ?

Reply:
updated documentation after checking with @abperiasamy